src/new/wallet.js

```diff
@@ -874,6 +886,7 @@ class HathorWallet extends EventEmitter {
         tokenHistory.sort((elem1, elem2) => elem2.timestamp - elem1.timestamp);
       } else {
         const currentHistory = tokensHistory[tokenUid];
+        console.log('will try to find index');
```

Comment:
Debug log

Reply:
Sorry for that, removed in 5bd097eed520f8eafcaf9e559ad40b74f869a6d3